### PR TITLE
fix: linux installation instructions update

### PR DESCRIPTION
**Pull request by mbergo**
This fixes the current way of install Jenkins X on Linux, working with Fedora 37 and ZSH

Signed-off-by: Marcus Bergo <marcus.bergo@gmail.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist:

- [x] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [x] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [x] Any dependent changes have already been merged.

/assign @rawlingsj

### CODE REVIEW

File: content/en/docs/_index.html [hunk 1]
```diff
@@ -171,11 +171,14 @@ <h3>Using Curl</h3>
...
 {{<highlight sh>}}
-# Download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
-curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent "https://github.com/jenkins-x/jx/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+# Download the `.tar` file, and unarchive it in ``a directory where you can run the `jx` command.
+curl -LO https://github.com/jenkins-x/jx/releases/download/latest/jx-linux-amd64.tar.gz -LO https://github.com/jenkins-x/jx/releases/download/latest/jx-linux-amd64.tar.gz.sig
 
-# Move binary to your path
-sudo mv jx /usr/local/bin
+# Verify the integrity of the d
+cosign verify-blob --key https://raw.githubusercontent.com/jenkins-x/jx/main/jx.pub --signature jx-linux-amd64.tar.gz.sig jx-linux-amd64.tar.gz
```
Comment (ankitm123) [inline content/en/docs/_index.html:178]: This needs to be changed as we moved to keyless signing. Going to verify that it works, and comment here.

**Review by tomhobson — APPROVED** [review]
Thanks for the PR

File: content/en/docs/_index.html (hunk 1 above)
Comment (tomhobson) [reply content/en/docs/_index.html:178]: Looks like we never got around to it, but it's currently borked so this is better than that.

**Pull request merged**